lib.rs

```diff
@@ -238,6 +245,18 @@ impl<T: ?Sized> Arc<T> {
     }
 }
 
+// `no_std`-compatible abort by forcing a panic while already panicing.
+fn abort() -> ! {
```

Comment:
Can we conditionally cfg this; when we have std we use the regular abort?

Reply:
Sure.